packages/shipjs-lib/src/lib/config/loadConfig.js

```diff
@@ -1,13 +1,30 @@
 import dotenv from 'dotenv';
 import { resolve } from 'path';
-import { existsSync } from 'fs';
+import { promises, constants } from 'fs';
 
 import defaultConfig from './defaultConfig';
 import mergeConfig from './mergeConfig';
 dotenv.config();
 
-export default function loadConfig(dir = '.', filename = 'ship.config.js') {
-  const fullPath = resolve(dir, filename);
-  const userConfig = existsSync(fullPath) ? require(fullPath) : {};
+const exist = path =>
+  promises
+    .access(path, constants.F_OK)
+    .then(() => path)
```

Comment:
```suggestion
    .then(() => true)
```

both are truthy, but I think exists should return a boolean

Reply:
no, i use `path` in next operation chain